downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -30,6 +30,186 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ### New features
 
+#### Introduction of several features to workers
+
+Several new features have been introduced to enhance the capabilities of workers.  
+
+- Introduction of alternate receive action
+
+The alternate receive action can be used to receive values from multiple send actions. It operates by waiting until it encounters a non-error message, a panic termination status on a closed channel, or the closure of all channels. Alternate receive action sets the first non-error value it encounters as the outcome.
+
+```ballerina
+import ballerina/io;
+import ballerina/lang.runtime;
+
+public function main() {
+    worker w1 {
+        2 -> w3;
+    }
+
+    worker w2 {
+        runtime:sleep(2);
+        3 -> w3;
+    }
+
+    worker w3 returns int {
+        // The value of the variable `result` is set as soon as the values from either
+        // worker `w1` or `w2` is received.
+        int result = <- w1 | w2;
+        return result;
+    }
+
+    worker w4 returns error? {
+        int value = 10;
+        if value == 10 {
+            return error("Error in worker 4");
+        }
+        value -> w6;
+    }
+
+    worker w5 {
+        runtime:sleep(2);
+        3 -> w6;
+    }
+
+    worker w6 returns int|error? {
+        // Alternate receive action waits until a message that is not an error is received. 
+        // Since `w4` returns an error, it waits further and sets the value that is received from `w5`.
+        int a = check <- w4 | w5;
+        return a;
+    }
+
+    int w3Result = wait w3;
+    io:println(w3Result);
+
```

Comment:
Shall we move this closer to the usage?

Reply:
Removed the unnecessary workers and simplified the code. However, can't move them closer as the named workers have to be defined at the beginning of the [block-function-body](https://ballerina.io/spec/lang/master/#block-function-body)